<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>761</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chartLayout"/>
   </widget>
   <widget class="QGroupBox" name="faimsconfiguregroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>FAIMS Configure</string>
    </property>
    <widget class="QGroupBox" name="workmodegroupBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Work Mode</string>
     </property>
     <widget class="QCheckBox" name="mode1checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RealTime Mode</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="mode2checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Processing</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mode3checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>147</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Keithley Mode</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="sctvaluegroupBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>110</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>SCT_Value</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="slabel">
         <property name="text">
          <string>Vol span:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="svaluelineEdit">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>13</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>61</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="clabel">
         <property name="text">
          <string>Density:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cvaluelineEdit">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>91</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="tlabel">
         <property name="text">
          <string>Step size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tvaluelineEdit">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="datareceivegroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>161</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Data Receive Area</string>
     </property>
     <widget class="QTextEdit" name="datareceivetextEdit">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>32</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>168</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="startpushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearpushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>11</y>
      <width>57</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="ymaxlineEdit">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>348</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="yminlineEdit">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>761</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="xminlineEdit">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>598</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="xmaxlineEdit">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_file"/>
    <addaction name="actionSave_chart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSave_chart">
   <property name="text">
    <string>Save chart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
